physics/GFS_PBL_generic_pre.meta

```diff
@@ -280,6 +280,13 @@
   dimensions = ()
   type = logical
   intent = in
+[mraerosol]
+  standard_name = flag_for_merra2_aerosol_aware_for_thompson
```

Comment:
I realize that maybe this functionality only works for Thompson right now, but is it conceivable that it could be used with other schemes? I would avoid including `for_thompson` unless absolutely necessary. Also, I think that the latest standard name rules prefer `do_X` over `flag_for_X`.

Reply:
I believe it could be used with other schemes in the future. I will change the name to "do_merra2_aerosol_awareness"